src/plugins/Identifiers.js

```diff
@@ -90,10 +90,17 @@ module.exports = class Identifiers extends OverlayPlugin {
         this.androidInput = textInput.createTextInput({
             regexFilter: new RegExp(`^[${HEX}]{0,16}$`),
             regexValidField: new RegExp(`^[${HEX}]{16}$`),
+            placeholder: 'e.g. 2b76129a48d5eb49',
             classes: 'gm-identifiers-android-input',
+            messageField: true,
             onChange: () => {
                 this.container.classList.remove('gm-identifiers-saved');
                 this.checkIDsValidity();
+                if (!this.androidInput.checkValidity()) {
```

Comment:
Could add to the commit message that validation has been added to the android identifier field.

It would help later when viewing the git log, to understand the changes in the commit.